x-pack/plugins/ml/server/models/bucket_span_estimator/single_series_checker.js

```diff
@@ -14,301 +14,299 @@
 import { mlLog } from '../../lib/log';
 import { INTERVALS, LONG_INTERVALS } from './intervals';
 
-export function singleSeriesCheckerFactory({ asCurrentUser }) {
-  const REF_DATA_INTERVAL = { name: '1h', ms: 3600000 };
+const REF_DATA_INTERVAL = { name: '1h', ms: 3600000 };
+
+export class SingleSeriesChecker {
```

Comment:
Removes `singleSeriesCheckerFactory` provider function and just exports the class.
`asCurrentUser` is now passed into the constructor.